app/classifier/tasks/survey/annotation-view.jsx

```diff
@@ -0,0 +1,74 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+class AnnotationView extends React.Component {
```

Comment:
This component has no state so can actually be written as a function `function AnnotationView(props) {…}`. The event handlers can just be named functions declared within the `AnnotationView` function. See the [Draggable](https://github.com/zooniverse/Panoptes-Front-End/blob/0fada40c695d9f871df2e6094ff89e0ecd17e187/app/lib/draggable.jsx) component for an example of event handlers in a functional component.